### ISSUE TITLE
Alarm-notify with var not replaced

### ISSUE THREAD

**Issue by EnzoAkira** [netdata/netdata#8504]
##### Bug report summary

I created a SRE role in the alarms most important to me, but in Alerta.io I am receiving all the roles.
I checked alarm-notify.sh and saw that some default Netdata paths are not correct, they still have the variables that would be replaced.

##### OS / Environment

```
centos-6 & 7
```

##### Netdata version

 Netdata 1.20

##### Component Name

Netdata alarm notifications

##### Steps To Reproduce

- In alarms, add the `sre` role to the `to:` line.
- Configure all settings for Alert.io at health_alarm_notify.conf
- Comment all `role_recipients_alerta`
- Just add `role_recipients_alerta[sre]="${DEFAULT_RECIPIENT_ALERTA}"`

##### Expected behavior

Receive alarms only the SRE role

**Comment by EnzoAkira** [comment]
Aditional Info:
![imagem](https://user-images.githubusercontent.com/39481527/77678605-52e28100-6f89-11ea-9021-05d0101b7f6d.png)

**Comment by ilyam8** [comment]
Hi @EnzoAkira 

How did you install Netdata?

I installed in from source and i have no such problem

```sh
[ilyam@ilyam-pc plugins.d]$ awk 'NR >= 190 && NR <= 196' alarm-notify.sh
# defaults to allow running this script by hand

[ -z "${NETDATA_USER_CONFIG_DIR}" ] && NETDATA_USER_CONFIG_DIR="/opt/netdata/etc/netdata"
[ -z "${NETDATA_STOCK_CONFIG_DIR}" ] && NETDATA_STOCK_CONFIG_DIR="/opt/netdata/usr/lib/netdata/conf.d"
[ -z "${NETDATA_CACHE_DIR}" ] && NETDATA_CACHE_DIR="/opt/netdata/var/cache/netdata"
[ -z "${NETDATA_REGISTRY_URL}" ] && NETDATA_REGISTRY_URL="https://registry.my-netdata.io"
[ -z "${NETDATA_REGISTRY_CLOUD_BASE_URL}" ] && NETDATA_REGISTRY_CLOUD_BASE_URL="https://netdata.cloud"
```

**Comment by EnzoAkira** [comment]
I'm using an ansible-playbook that runs `sh /tmp/kickstart-static64.sh --dont-wait --stable-channel`

**Comment by ilyam8** [comment]
I dont have such problem using `bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh)` to install Netdata.

**Comment by ilyam8** [comment]
Could be that you are checking source code?

**Comment by EnzoAkira** [comment]
I'm not.
I updated netdata on all hosts and implemented the settings for Alerta.io. The example I gave above is of a netdata installation on a host in production.

**Comment by ilyam8** [comment]
@prologic @Ferroin any ideas?

**Comment by EnzoAkira** [comment]
By the way. What script replace this vars? netdata-installer or kickstart-static64?

**Comment by prologic** [comment]
I will take a look today...

**Comment by prologic** [comment]
> By the way. What script replace this vars? netdata-installer or kickstart-static64?

That would be `build/subs.inc`:

```sh
$ sift '.*-e.*@.*_POST@'
build/subst.inc:3:		-e 's#[@]localstatedir_POST@#$(localstatedir)#g' \
build/subst.inc:4:		-e 's#[@]sbindir_POST@#$(sbindir)#g' \
build/subst.inc:5:		-e 's#[@]pluginsdir_POST@#$(pluginsdir)#g' \
build/subst.inc:6:		-e 's#[@]configdir_POST@#$(configdir)#g' \
build/subst.inc:7:		-e 's#[@]libconfigdir_POST@#$(libconfigdir)#g' \
build/subst.inc:8:		-e 's#[@]cachedir_POST@#$(cachedir)#g' \
build/subst.inc:9:		-e 's#[@]registrydir_POST@#$(registrydir)#g' \
build/subst.inc:10:		-e 's#[@]varlibdir_POST@#$(varlibdir)#g' \
```

This gets runs on every build which also includes the static `kickstart_static64.sh` installer which uses a prebuilt makeself bundle. So the only way I can _think_ of this not working correctly is either:

- Your version of `sed` on your systems is causing this to break and fail silently
- You are building from source and don't have `sed` installed

But none of this makes sense as you would see significant problems elsewhere with templated paths not substituted correctly and probably half of NetData just wouldn't function at all.

I also cannot repro this at all on a source-based install, kickstart-based install nor do I see it on any of my production systems either :/

**Comment by prologic** [comment]
@EnzoAkira At this point I need some help from you to root cause this. Would it be possible to get a copy of the tooling/scripts that are driving your installs? Or some more details based on my comments? I've gone through every comment and detail you've provided thus far and nothing is making any sense as to what the cause _might_ be nor can I repro this (yet).

**Comment by EnzoAkira** [comment]
Hello.
I found the problem and it is on my side.
I reviewed all of my playbook that installs netdata and found that I was copying the original 'alarm-notify' file.
I did this a while ago when I was testing Alerta.io because netdata was inserting in the 'origin' field the prefix 'netdata/${host}', so I removed the prefix.
Now I just edit this line and I don't copy the complete file.

Thanks so much for your help!!!
